docs/upgrading.md

```diff
@@ -5,6 +5,11 @@ the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summar
 
 ## Upgrading to v3.6
 
+### JSON templating fix
```

Comment:
I'd probably move this near the legacy `patch` `pod` removal if the thought is that it shouldn't have much impact. I.e. order of importance (then all the metrics changes as things after it could be missed accidentally since it's a large section)

Reply:
I've moved it to the bottom as least important.